mainflux/users.py

```diff
@@ -7,44 +7,64 @@
 
 
 class Users:
-    users_endpoint = "users"
+    USERS_ENDPOINT = "users"
 
     def __init__(self, url: str):
-        self.url = url
+        self.URL = url
 
-    def create(self, user: dict):
+    def create(self, user: dict, token: str = ""):
         """Registers new user account given email and password. New account
         will be uniquely identified by its email address."""
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/users", json=user)
+        http_resp = requests.post(
+            self.URL + "/users",
+            json=user,
+            headers=utils.construct_header(token, utils.CTJSON),
+        )
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
             mf_resp.error.message = errors.handle_error(
                 errors.users["create"], http_resp.status_code
             )
         else:
-            location = http_resp.headers.get("location")
-            mf_resp.value = location.split("/")[2]
+            mf_resp.value = http_resp.json()
         return mf_resp
 
     def login(self, user: dict):
         """Generates an access token when provided with proper credentials."""
         mf_resp = response.Response()
-        http_resp = requests.post(self.url + "/tokens", json=user)
+        http_resp = requests.post(self.URL + "/users/tokens/issue", json=user)
         if http_resp.status_code != 201:
             mf_resp.error.status = 1
             mf_resp.error.message = errors.handle_error(
                 errors.users["login"], http_resp.status_code
             )
         else:
-            mf_resp.value = http_resp.json()["token"]
+            mf_resp.value = http_resp.json()
+        return mf_resp
+    
+    def refresh_token(self, user: dict, token: str):
```

Comment:
Refresh token doesn't take user as a parameter
```suggestion
    def refresh_token(self, refresh_token: str):
```